tests/test_model.py

```diff
@@ -174,6 +175,32 @@ def test_component_cpes():
         assert unused_variant.cpe not in cpes
 
 
+@pytest.mark.django_db(databases=("default", "read_only"), transaction=True)
+def test_component_without_variant_cpes():
```

Comment:
Let's add a docstring comment:
```suggestion
@pytest.mark.django_db(databases=("default", "read_only"), transaction=True)
def test_component_without_variant_cpes():
    """Test the CPEs property on the Component model finds CPEs
        even when no Variants are linked directly to that Component"""
```

Reply:
ok